java/acceptancetest/features/web/WebPanel.feature

```diff
@@ -8,15 +8,13 @@ Scenario Outline: Web Panel requests
    When I ask for the url <panelURL>
    Then either <PageTitle> or <FormattedPageTitle> is returned as the title
 
-   # firefox version of the test appears to be hanging on travis
-   @firefox @Ignore
+   @firefox
    Examples: Firefox Panel Tests
    | browser | panel | panelURL | PageTitle | FormattedPageTitle |
    | firefox | java/test/jmri/jmrit/cabsignals/SimpleCabSignalTestPanel.xml | http://localhost:12080/panel/Layout/Cab%20Signal%20Test  | Layout/Cab%20Signal%20Test \| My JMRI Railroad |Cab Signal Test \| My JMRI Railroad|
    | firefox | java/test/jmri/jmrit/display/layoutEditor/load/LayoutEditorTest-4-19-4.xml | http://localhost:12080/panel/Layout/Layout%20Editor%20Test  | Layout/Layout%20Editor%20Test \| My JMRI Railroad | Layout Editor Test \| My JMRI Railroad |
    | firefox | java/test/jmri/jmrit/display/layoutEditor/load/Decorations-4-19-6.xml | http://localhost:12080/panel/Layout/Decorations%20Testing  | Layout/Decorations%20Testing \| My JMRI Railroad | Decorations Testing \| My JMRI Railroad |
 
-   # Chrome version failing on travis
    @chrome @Ignore
```

Comment:
In the other files, both the comment and the `@Ignore` statement is removed. In this file, only the comment is removed, but not the `@Ignore` statement. Is this intentional? If the `@Ignore` statement should be here, I think it's good to keep the comment as well.